index.js

```diff
@@ -106,3 +107,16 @@ btnBack.addEventListener('click', function () {
         console.log(nr2);
     }
 });
+
+
+btnHistory.addEventListener("click", function () {
+    const initialText = 'Show History';
+    calcHistory.classList.toggle('hidden');
+
+    if (btnHistory.textContent.toLowerCase().includes(initialText.toLowerCase())) {
+        btnHistory.textContent = 'Hide History';
+    } else {
+        btnHistory.textContent = initialText;
+    }
+});
```

Comment:
Let's add a `isHistoryToggled` variable and decide what text to show based on it. Something like this:

```
let isHistoryToggled = false;


btnHistory.addEventListener("click", function () {
    isHistoryToggled = !isHistoryToggled;
    calcHistory.classList.toggle('hidden');


    if (???) {
        btnHistory.textContent = 'Hide History';
    } else {
        btnHistory.textContent = 'Show History';
    }
});
```